src/main/kotlin/org/eclipse/lmos/operator/server/routing/CustomResourcesController.kt

```diff
@@ -26,14 +26,14 @@ private const val KUBERNETES_DEFAULT_NAMESPACE = "default"
 class CustomResourcesController(private val customResourcesService: CustomResourcesService) {
     @GetMapping("/channels")
     fun getChannels(
-        @RequestHeader(name = X_SUBSET_HEADER) subsetHeader: String,
+        @RequestHeader(name = X_SUBSET_HEADER, required = false) subsetHeader: String,
```

Comment:
If we set `required` to false, should we also change the type of `subsetHeader`to `String?` to make it nullable? Or will spring somehow take care that we don't get a NPE here?